tap_trello/streams.py

```diff
@@ -32,7 +32,7 @@ def check_order(self, current_value):
             gt_lt = "less than" if self.order == 'ASC' else "greater than"
             raise Exception(
                 "Detected out of order data. In {} sorted stream, current sorted " +
-                "value {} is {} last sorted value {}".format(
+                "value {} is {} last sorted value {}".format( # pylint: disable=too-many-format-args
```

Comment:
This is actually an issue, the code should probably be this (so the format hits all of the `{}` values:
```
("Detected out of order data. In {} sorted stream, current sorted " +
                "value {} is {} last sorted value {}").format(	                "value {} is {} last sorted value {}".format(
```